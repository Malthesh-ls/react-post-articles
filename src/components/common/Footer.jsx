function Footer() {
  const currentDate = new Date();

  return (
    <div className="footer">
      &copy;
      {currentDate.getFullYear()}
      {' '}
      - React Post Articles
    </div>
  );
}

export default Footer;
