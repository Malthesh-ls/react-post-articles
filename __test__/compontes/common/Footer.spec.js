import renderer from 'react-test-renderer';
import Footer from '../../../src/components/common/Footer';

describe('Footer Tests', () => {
  it('should match snapshot', () => {
    const snapshot = renderer
      .create(<Footer />)
      .toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
