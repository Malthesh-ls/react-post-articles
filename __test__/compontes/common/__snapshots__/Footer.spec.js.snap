// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Tests should match snapshot 1`] = `
<div
  className="footer"
>
  Â©
  2024
   
  - React Post Articles
</div>
`;
